SDC_DB/postgresDB/seedDB.sql

```diff
@@ -0,0 +1,51 @@
+DROP DATABASE silverspoon;
+
+CREATE DATABASE silverspoon;
+
+\c silverspoon;
+
+BEGIN;
+
+CREATE TABLE restaurants(
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(50) NOT NULL
+);
+
+\copy restaurants (name) FROM '/Users/m.azarshahy/Desktop/SDC/menu-components/SDC_DB/postgresDB/testData/names.csv' DELIMITER ',' CSV;
+
+COMMIT;
+
+BEGIN;
+
+CREATE TABLE items(
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(100) NOT NULL,
+  menu VARCHAR(10) NOT NULL,
+  price INT NOT NULL,
+  res_id INT NOT NULL
+);
+
+\copy items (name, menu, price, res_id) FROM '/Users/m.azarshahy/Desktop/SDC/menu-components/SDC_DB/postgresDB/testData/items.csv' DELIMITER ',' CSV;
+
+COMMIT;
+
+BEGIN;
+
+CREATE TABLE tags(
+  id SERIAL PRIMARY KEY NOT NULL,
```

Comment:
To help speed up seeding you could try adding the primary key constraint after your data has been seeded